stack_definition.json

```diff
@@ -415,7 +415,67 @@
 				}
 			],
 			"name": "6 - WatsonX SaaS services",
-			"version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.a8504fac-eb37-4aa0-a284-f26e876b5750-global"
+			"version_locator": "1082e7d2-5e2f-0a11-a3bc-f88a8e1931fc.9932d3b1-369e-4b5c-ad39-9e10149b83b0-global"
+		},
+		{
+			"inputs": [
+				{
+					"name": "use_existing_resource_group",
+					"value": true
+				},
+				{
+					"name": "resource_group_name",
+					"value": "ref:../../members/1%20-%20Account%20Infrastructure%20Base/outputs/workload_resource_group_name"
+				},
+				{
+					"name": "region",
+					"value": "ref:../../inputs/region"
+				},
+				{
+					"name": "prefix",
+					"value": "ref:../../inputs/prefix"
+				},
+				{
+					"name": "plan",
+					"value": "platinum"
+				},
+				{
+					"name": "elasticsearch_version",
+					"value": "8.12"
+				},
+				{
+					"name": "existing_kms_instance_crn",
+					"value": "ref:../../members/2a%20-%20Security%20Service%20-%20Key%20Management/outputs/kms_instance_crn"
+				},
+				{
+					"name": "kms_endpoint_type",
+					"value": "private"
+				},
+				{
+					"name": "member_host_flavor",
+					"value": "b3c.4x16.encrypted"
+				}
+			],	
+			"name": "7 - Databases for Elasticsearch",
+			"version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.c5ea982b-011a-46c8-a49d-0de273f40b46-global"
+		}
+	],
+	"outputs": [
+		{
+		  "name": "es_hostname",
+		  "value": "ref:./members/7 - Databases for Elasticsearch/outputs/hostname"
```

Comment:
lets be consistent with url encoding. we either us it or not. It seems everywhere we are url encoding, but its not longer needed.